Server/Services/DataService.cs

```diff
@@ -2133,5 +2155,24 @@ private string[] FilterUsersByDevicePermissionInternal(AppDb dbContext, IEnumera
                 .Select(x => x.Id)
                 .ToArray();
         }
+
+        private AppDb GetDbContext()
```

Comment:
I'm curious, what's the impetus for switching to this instead of `IDbContextFactory`?

Reply:
After the update, it threw an exception when trying to cast `IDbContextFactory<SqliteDbContext>` (or any of the derived DB contexts) to `IDbContextFactory<AppDb>`.

There may be another way to accomplish it, maybe by implementing our own factory.  Or maybe I was doing it wrong from the start?  But since the DbContext itself doesn't need any injected dependencies, this seemed like the path of least resistance.

Reply:
Yeah, seems to work fine if we create our own factory:  https://github.com/immense/Remotely/tree/app-db-factory

Reply:
@cmbankester  By the way, thanks for reviewing and commenting.  :)  It's really cool to be finally interacting with you guys!

Reply:
Of course! I'm more than happy to do some code reviews on this project - your code is delightfully clean